content/guides/figma/getting-started.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Getting started
+---
+import {LinkExternalIcon} from '@primer/octicons-react'
+
+## What are Primer Figma libraries?
+The Primer Figma libraries contain UI components & design tokens (styles) that our team’s at GitHub use to design GitHub. The components contained within Primer match what is available for developers in Primer React Components, Primer ViewComponents, and Primer CSS.
+
+## Installation
+To use a library in Figma enable (install) it from the **assets** tab (`option` + `2`).
```

Comment:
Do we want to mention that some libraries are enabled by default at the workspace level?

Reply:
I am not sure that is helpful. I think it may be more confusing for users.